src/components/CommonMessage/CommonMessage.jsx

```diff
@@ -27,7 +27,7 @@ const compileMessage = (props) => {
 
     return (
       <div
-        className={ 'c-common-message__background ' + messageType }
+        className={ props.location === 'header' ? 'c-common-message__background ' + messageType :'c-common-message__background small ' + messageType }
```

Comment:
this component follows bemit for example as a reference within the reskin of the gui wallet. _small_ should be _c-common-message__background--small_